src/hooks/rewards/util.ts

```diff
@@ -0,0 +1,28 @@
+export function pointsToEstimatedDydxRewards(
+  points?: number,
+  totalPoints?: number,
+  dydxPrice?: number,
+  totalUsdRewards?: number
+) {
+  if (!totalPoints || !dydxPrice || !totalUsdRewards || points === undefined) return '-';
+  const usdRewards = (points / totalPoints) * totalUsdRewards;
+  return usdRewards / dydxPrice;
+}
+
+export function pointsToEstimatedDollarRewards(
+  points?: number,
+  totalPoints?: number,
+  totalUsdRewards?: number
+) {
+  if (!totalPoints || !totalUsdRewards || points === undefined) return undefined;
+  return (points / totalPoints) * totalUsdRewards;
+}
+
+// Move to Chaos Labs query once its available
+export const SEPT_2025_REWARDS_DETAILS = {
```

Comment:
does this need to be moved? or should we just remove the comment

Reply:
it still needs to come from the BE